### PR TITLE
Implement the video in the admin UI

### PR DESCRIPTION
**Pull request by Coenego**
screen shot 2015-02-05 at 09 38 34

### CODE REVIEW

**Comment by Coenego** [comment]
Assigning to @bp323 for review

File: apps/timetable/admin/ui/js/index.js [hunk 1]
```diff
@@ -273,6 +273,8 @@ define(['gh.core', 'gh.admin-constants', 'gh.subheader', 'gh.calendar', 'gh.admi
         $('#gh-main-tripos .gh-video:first-child').hide();
         // Do not show the video next time
         gh.api.utilAPI.localDataStorage().store('hideVideo', true);
+        // Start the video
```
Comment (bp323) [inline apps/timetable/admin/ui/js/index.js]: `Stop`

File: shared/gh/js/views/gh.video.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    //////////////////////
+    //  INITIALISATION  //
+    //////////////////////
+
+    /**
+     * Initialise the video component
+     *
+     * @private
+     */
+    var initialise = function() {
```
Comment (bp323) [inline shared/gh/js/views/gh.video.js]: We usually name these something like `initialiseVideo`

**Comment by bp323** [comment]
Code looks good. In Chrome I can't see the video but hear the audio, Firefox doesn't play the video at all because it doesn't support it. Could you have a look and see if native HTML will be enough to work in all browsers?

**Comment by Coenego** [comment]
Reassigning to @bp323 for review after follow-up

**Comment by bp323** [comment]
Verified and merged

**Pull request merged**